### PR TITLE
Fixed typo and format issues in README.md

### PR DESCRIPTION
**Pull request by harishp8889**
# Description
Fixed typo and format issues in README.md

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Viewed in-branch webpage

### CODE REVIEW

**Review by mjsdell — APPROVED** [review]
LGTM

**Review by harshitap26 — APPROVED** [review]

**Review by harshitap26 — COMMENTED** [review]
The checks are failing

**Comment by harishp8889** [comment]
> The checks are failing

Those checks will be fixed in the coming PRs as it requires changes in many places.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -20,4 +20,5 @@
  
 
 # for all files:
-* @kumarp20 @bpjain2004 @francis-nijay @harishp8889 @spriya-m @prablr79 @hoppea2 @tdawe @coulof @dattaarindam
+* @hoppea2 @dattaarindam @bpjain2004 @francis-nijay @harishp8889 @kumarp20 @prablr79 @spriya-m @tdawe @coulof
```
Comment (spriya-m) [inline .github/CODEOWNERS:23]: @harishp8889 , could you please correct this?

**Review by spriya-m — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (harishp8889) [reply .github/CODEOWNERS:23]: @spriya-m I have corrected the error, The issue is `dattaarindam` this user is not added as part of this repo and hence still the error is thrown. @prablr79 Can you please add him to this repo

**Review by francis-nijay — APPROVED** [review]

**Review by spriya-m — APPROVED** [review]

**Pull request merged**